docs/services/data/_index.md

```diff
@@ -19,6 +19,12 @@ The service is designed for flexibility and efficiency while preventing data los
 
 The service has two parts: [Data Capture](#data-capture) and [Cloud Sync](#cloud-sync).
 
+## Required Components
+
+{{< cards >}}
```

Comment:
This is correct but not helpful. In this case the components it works for would be more useful